### DISCUSSION TITLE
An error occurred [undefined] Unauthorized when dropdown opened

### DISCUSSION THREAD

**Discussion by mastertIT** [kimai/kimai#4425] [On-Premise: Installation & Updates] [answered]
when selecting the project from the dropdpown upon creating a new timesheet item the following error shows up

> 
> > An error occurred
> [undefined] Unauthorized

preceding this error a login popup shows up.

system is behind a reverse proxy running on another vm.
trusted_proxy, trusted_host, cors_allow_origin have been setup

![image](https://github.com/kimai/kimai/assets/94576286/f5c794b9-a2f3-4786-a69d-39ad26a4ab44)

![image](https://github.com/kimai/kimai/assets/94576286/10c9ea01-2765-4e13-9544-65fb707ec53d)

**Comment by kevinpapst** [comment]
This obviously does not just happen by using the project dropdown. Provide more infos please.

Does it always happen? 
Did you log-out or switch the user in another tab?
Was the screen open for days?

**Reply by mastertIT** [reply]
This only happens when using the dropdowns in the timesheet menu. 

Normal use in the setup and admin screens doesn't cause this.

Is there anything needed for the reverse proxy to work besides the 3 variables l initially listed?

**Reply by kevinpapst** [reply]
Sounds like a file permission problem where the cache folder is not writable.
Check Doctor > Logs

**Reply by mastertIT** [reply]
Getting green checkmarks for /var/cache as well as /var/log, /var/data

**Reply by kevinpapst** [reply]
Then some other strange error happens. 

- Check your logs
- Check your browser console => network tab and post the results of the failed call

Which version is this anyway? The Sign-In window is not from Kimai! 
Maybe there is another error in your network setup.

**Reply by mastertIT** [reply]
This is latest version running as docker apache, with a kemp reverse proxy.
Something to note is I see under the doctor tab that there is an error for the log file which states missing log file. 

I'll check other logs, which one's would be suggested? 
My assumption is an api call is being requested when the dropdown is selected but uses an unknown cookie / authentication format which results in chrome defaulting to a login request. 

Will provide network tab screenshot as well as other logs once I have a chance.

**Reply by mastertIT** [reply, marked as answer]
We've determined that a content rule on our proxy was redirecting the traffic to another application. 
the reason this login dialog opens is because the url requested upon selecting the dropdown was  api/activities?project=xxxxxxx
our content rule was matching on /api and forwarding it. we were using a host rule to direct all traffic to kimai when it's fqdn is matched. we've modified our content rule to explicitly require the fqdn & /api to be called in the same regex statement
